aospy/utils/times.py

```diff
@@ -115,7 +115,9 @@ def monthly_mean_ts(arr):
     monthly_mean_at_each_ind : Copy monthly means to each submonthly time
 
     """
-    return arr.resample(**{TIME_STR: '1M'}).mean(TIME_STR).dropna(TIME_STR)
+    return arr.resample(
+        **{TIME_STR: '1M'}, restore_coord_dims=True
```

Comment:
Well that's annoying...clearly not invalid syntax.

Reply:
I think this might be because the `flake8` check stickler uses is [by default for Python 2](https://stickler-ci.com/docs#python).  Maybe if we update the configuration in `stickler.yml` it will fix this?
```
linters:
  flake8:
    fixer: true
    python: 3
```

Reply:
That did the trick!  Thanks.